scans/port.py

```diff
@@ -0,0 +1,15 @@
+import socket
+
+
+class Port:
+    def __init__(self, port):
+        self.port = port
+
+    def valid(self):
+        if not isinstance(self.port, int):
+            return False
+
+        if self.port == 22:
```

Comment:
You could do something like:
```
return self.port == 22
```

But the way you have it now, makes it easier to add another check.

Reply:
I'm cool with that, I doubt we'll be making much changes here.  More of a function of copy/paste than preference and I like that it's more brief.  Optimizing for the future is an anti-pattern in this context, so I will implement your suggestion.

Reply:
Fixed in bf4d857